Sources/PrivacyDashboard/PrivacyDashboardController.swift

```diff
@@ -244,7 +244,11 @@ extension PrivacyDashboardController: WKNavigationDelegate {
         privacyInfo?.$serverTrust
             .receive(on: DispatchQueue.global(qos: .userInitiated))
             .map { serverTrust in
-                ServerTrustViewModel(serverTrust: serverTrust)
+                if let serverTrust {
+                    // swiftlint:disable:next force_cast
+                    return ServerTrustViewModel(serverTrust: (serverTrust as! SecTrust))
+                }
+                return ServerTrustViewModel(serverTrust: nil)
```

Comment:
I am unsure why we are doing this?

Reply:
so we can properly mock the secTrust in tests, I couldn't get away without warning or error hence this code, if you know a better way I'll be happy to simplify it!